### PR TITLE
concepts completion

### PR DESCRIPTION
**Pull request by bastienbeurier**
- Remove webhooks ref
- Add webhooks & templates concepts to docs

## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: docs-v2/integrate/guides/authorize-an-api.mdx [hunk 1]
```diff
@@ -144,46 +142,40 @@ nango
...
+2. specify a _Webhook URL_ to which Nango will send notifications
+3. listen for webhooks in your backend at the specified route
+4. enable the "Send New Connection Creation Webhooks" checkbox
+
+Nango webhooks are post requests with the following JSON body: 
+```json
+{
+    "type": "auth", 
+    "connectionId": "<CONNECTION-ID>",
+    "authMode": "OAUTH2|API_KEY|BASIC|OAUTH1|APP",
```
Comment (khaliqgant) [inline docs-v2/integrate/guides/authorize-an-api.mdx]:
```
export enum AuthModes {
    OAuth1 = 'OAUTH1',
    OAuth2 = 'OAUTH2',
    Basic = 'BASIC',
    ApiKey = 'API_KEY',
    AppStore = 'APP_STORE',
    Custom = 'CUSTOM',
    App = 'APP',
    None = 'NONE'
}
```

File: docs-v2/integrate/guides/receive-webhooks-from-an-api.mdx [hunk 2]
```diff
@@ -4,27 +4,32 @@ sidebarTitle: 'Receive webhooks from an API'
...
 
-Webhooks received by Nango from external APIs are immediately forwarded to your backend ([step-by-step guide](/integrate/guides/receive-webhooks-from-nango) to receive webhooks from Nango).
+Nango automatically forwards all webhooks from external APIs to your app. 
 
-Webhook forwarding is automatically enabled once you send webhooks from external APIs to Nango.  
-
-Here is the body of a webhook from Nango forwarding a webhook from an API: 
+Nango webhook forwards are POST requests with the following JSON body:
 ```json
 {
```
Comment (khaliqgant) [inline docs-v2/integrate/guides/receive-webhooks-from-an-api.mdx:34]: Also has 

```
            connectionId,
            providerConfigKey
```

In some cases the connectionId may not be known if the validation fails or something goes wrong in identifying the webhook

**Review by khaliqgant — CHANGES_REQUESTED** [review]
Few comments inline

File: docs-v2/integrate/guides/authorize-an-api.mdx (hunk 1 above)
Comment (bastienbeurier) [reply docs-v2/integrate/guides/authorize-an-api.mdx]: Done!

File: docs-v2/integrate/guides/receive-webhooks-from-an-api.mdx (hunk 2 above)
Comment (bastienbeurier) [reply docs-v2/integrate/guides/receive-webhooks-from-an-api.mdx:34]: Done!

File: docs-v2/integrate/guides/advanced/secure-frontend-sdk.mdx [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+Generate the HMAC signature in your backend and pass it to your frontend before you make `nango.auth` calls.
+
+The HMAC signature can be generated with the following code:
+
+<Tabs>
+
+<Tab title="Javascript/Typescript">
+
+```js
+import * as crypto from 'node:crypto';
```
Comment (bodinsamuel) [inline docs-v2/integrate/guides/advanced/secure-frontend-sdk.mdx:20]:
```suggestion
import crypto from 'node:crypto';
```

**Pull request merged**